server/routes/user.js

```diff
@@ -32,7 +32,8 @@ router.post("/add", function (req, res, next) {
     id: "user",
     extension: [
       {
-        url: config.fhir.server + "/StructureDefinition/iHRISUserDetails",
+        //use this domain which is the only valid for the search parameters
+        url:config.profileUrlDomain + "/StructureDefinition/iHRISUserDetails",
```

Comment:
Why do we have a different config variable here? It should match config.fhir.server or config.structureDefinition